pages/keep-eth.tsx

```diff
@@ -29,23 +38,26 @@ function	EstimateGasRow(): ReactElement {
 }
 
 function	KeepEthPage(): ReactElement {
-	const	{keptEth, set_keptEth} = useFlow();
-	const	{balances, useWalletNonce} = useWallet();
+	const	router = useRouter();
+	const	{provider, isActive} = useWeb3();
+	const	{balances, updateWallet, useWalletNonce} = useWallet();
+	const	[isShowingArrow, set_isShowingArrow] = useState(false);
 	const	{yvEthData} = useYearn();
-	const	[isShowingArrow] = useState(false);
+	const	[keptEth, set_keptEth] = useState(ethers.constants.Zero);
```

Comment:
I was using LocalStorage to persist this across pages, how will it be persisted now?